components/gitpod-db/src/team-db.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the Gitpod Enterprise Source Code License,
+ * See License.enterprise.txt in the project root folder.
+ */
+
+import { Team } from "@gitpod/gitpod-protocol";
+
+export const TeamDB = Symbol('TeamDB');
+export interface TeamDB {
+    findTeamsByUser(userId: string): Promise<Team[]>;
```

Comment:
Potential improvements / follow-ups:
- create a new team
- delete a team
- add user to team
- remove user from team